apps/web/src/components/terminal/TerminalSearchBar.tsx

```
@@ -0,0 +1,113 @@
+          onNext();
+        }
+      }
+    },
+    [onClose, onNext, onPrevious]
+  );
+
+  return (
+    <div
+      className="absolute top-1 right-1 z-10 flex items-center gap-1 bg-card border border-border rounded-md px-2 py-1 shadow-lg"
+      onKeyDown={handleKeyDown}
+    >
```

Biome (2.3.13)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)